### ISSUE TITLE
Only one arrow is allowed to come in

### ISSUE THREAD

**Issue by adamralph** [Particular/ServiceInsight#707]
From RayGun.

Version 1.6.1+10.Branch.master.Sha.cdfac1d607bea941b09f0fd96c8fbaf482ff1675

```
ServiceInsight.SequenceDiagram.Diagram.Handler.set_In(Arrow value) in C:\BuildAgent\work\7540559e45164f93\src\ServiceInsight\SequenceDiagram\Diagram\Handler.cs:39
 ServiceInsight.SequenceDiagram.ModelCreator.Initialize() in C:\BuildAgent\work\7540559e45164f93\src\ServiceInsight\SequenceDiagram\ModelCreator.cs:99
 ServiceInsight.SequenceDiagram.SequenceDiagramViewModel.CreateElements(List`1 messages) in C:\BuildAgent\work\7540559e45164f93\src\ServiceInsight\SequenceDiagram\SequenceDiagramViewModel.cs:183
 ServiceInsight.SequenceDiagram.SequenceDiagramViewModel.Handle(SelectedMessageChanged message) in C:\BuildAgent\work\7540559e45164f93\src\ServiceInsight\SequenceDiagram\SequenceDiagramViewModel.cs:150
```

NServiceBus version 6.1.1
ServiceControl version 1.37.0

**Comment by distantcam** [comment]
This looks like a regression of #545

**Comment by HEskandari** [comment]
@distantcam Didn't we trap that exception and show the message/graphics as per #545? It looks like this is a duplicate and we should reopen #545.

**Comment by adamralph** [comment]
Discussed with @HEskandari. We will try and reproduce this in 1.6.1 (or later). Once that is done, we will also try and reproduce in 1.5.7 (which is the version in which we fixed #545).

If 1.5.7 does not contain the bug, then this is a regression, and we will fix the bug in this issue in the next release.

If 1.5.7 _does_ contain the bug, then we will re-open #545 and fix it in the next release, and close this issue as a duplicate.

**Comment by mauroservienti** [comment]
Cannot reproduce anymore.
